README.md

```diff
@@ -6,136 +6,160 @@
-options.LocalizedReasonBodyText = "REASON THAT APP WANTS TO USE BIOMETRY";
-options.LocalizedFallbackButtonText = "FALLBACK";
-options.LocalizedCancelButtonText = "CANCEL";
-
-// use LAPolicy.DeviceOwnerAuthenticationWithBiometrics for biometrics only with no fallback to passcode/password
-// use LAPolicy.DeviceOwnerAuthentication for biometrics+watch with fallback to passcode/password
-_biometryService = new BiometryService(options, async ct => "Biometrics_Confirm", LAPolicy.DeviceOwnerAuthentication);
-```
+Face authentication is only available when using `.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricWeak)` in the `BiometricPrompt.PromptInfo.Builder` instantiation that is required for the service. Please note that if you are using `.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricStrong)` in the `BiometricPrompt.PromptInfo.Builder` Face authentication is only available on a Google Pixel 4 as of now.
 
-#### Android
+Please note that Encrypt/Decrypt methods are only available when using `.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricStrong)` in the `BiometricPrompt.PromptInfo.Builder` instantiation that is required for the service.
 
-Face ID is only available with the followwing configuration `.SetAllowedAuthenticators(BiometricManager.Authenticators.BiometricWeak` during instantiation of the service.
+Please also note that the title and subtitle are used for `Fingerprint` and `Face` biometry.
 
-Encrypt/Decrypt method are only available with the following configuration `.SetAllowedAuthenticators(BiometricManager.Authenticators.BiometricStrong` during instantiation of the service.
+Here is an example of instantiation of the service for Android.
 
-An example of instantiation is as follow, with the fallback to the pin code with some text descriptions to display for the user.
+``` cs
+var promptBuilder = () => new BiometricPrompt.PromptInfo.Builder()
+	.SetTitle("Title")
+	.SetSubtitle("Subtitle")
+	.SetNegativeButtonText("Cancel")
+	.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricStrong)
+	.Build();
 
-```
-_biometryService = new BiometryService(
-    MainActivity.Instance,
-    CoreDispatcher.Main,
-    ct => Task.FromResult(new BiometricPrompt.PromptInfo.Builder()
-    .SetTitle("Biometrics SignIn")
-    .SetSubtitle("Biometrics Confirm")
-    .SetAllowedAuthenticators(BiometricManager.Authenticators.BiometricStrong | BiometricManager.Authenticators.DeviceCredential) // Fallback on secure pin
-    .SetNegativeButtonText("Cancel")
-    .Build()));
+var biometryService = new BiometryService(
+	fragmentActivity: MainActivity.Instance,
+	promptInfoBuilder: promptBuilder,
+	loggerFactory: null
+);
 ```
 
-## Methods
+#### iOS
 
-### GetGapabilites
+Please note that you must set `NSFaceIDUsageDescription` (key/value) in the `Info.plist` file otherwise the service may throw exceptions or fail.
```

Comment:
For that part of the sentence `the service may throw exceptions or fail.`, it's weirdly worded in my opinion. Like I don't see it where it will just fail here, won't it just throw no matter what

Reply:
I think it depends on the method. I'm not sure what is the behavior of the native code that we are using if it will throw any exception or just fail. But I know that in one or two methods we check if it's set and throw if it's not.